lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -141,9 +169,21 @@ spec = do
     sqliteSpecRnd
     loggingSpec
     connectionSpec
+    sqliteDetailedSeqSpec
+    sqliteDetailedRndSpec
```

Comment:
side-note: it'd be better to put the implementation of these tests in a separate module. This `SqliteSpec` was kept pretty simple as a top-level aggregator for all other specs. 
These tests should most probably leave in the `sqliteSpec` and sqliteSpecRnd modules.